app/models/profile_declaration.rb

```diff
@@ -1,14 +1,8 @@
 # frozen_string_literal: true
 
 class ProfileDeclaration < ApplicationRecord
-  delegated_type :declarable, types: %w[EarlyCareerTeacherProfile MentorProfile]
+  belongs_to :participant_profile
   belongs_to :participant_declaration
 
-  scope :uplift, lambda {
-                   joins(
-                     "left join early_career_teacher_profiles
-                      on declarable_type='EarlyCareerTeacherProfile'
-                      and declarable_id=early_career_teacher_profiles.id",
-                   ).merge(EarlyCareerTeacherProfile.uplift)
-                 }
+  scope :uplift, -> { where(participant_profile_id: ParticipantProfile::ECT.uplift.select(:id)) }
```

Comment:
https://github.com/DFE-Digital/early-careers-framework/pull/635/files#r663994053 probably needs some conflict resolving with this, right?

Reply:
Already resolved :)